graylog2-server/src/test/java/org/graylog/plugins/cef/parser/CEFMappingTest.java

```diff
@@ -34,4 +41,15 @@ public void forFullName() throws Exception {
             assertEquals(mapping, CEFMapping.forFullName(mapping.getFullName()));
         }
     }
-}
\ No newline at end of file
+
+    @Test
+    public void convertLargeValues() throws Exception {
+        String bigValueString = String.valueOf(Integer.MAX_VALUE) + 1;
+        BigInteger bigValue = new BigInteger(bigValueString);
+        for (String keyName : BIG_INT_FIELD_KEYS) {
+            CEFMapping fieldMapping = CEFMapping.forKeyName(keyName);
+            Object mapping = fieldMapping.convert(bigValueString);
```

Comment:
If you do create the array of CEFMapping, then this piece can just look like:

```suggestion
        for (CEFMapping fieldMapping : BIG_INT_FIELD_KEYS) {
            Object mapping = fieldMapping.convert(bigValueString);
```

Reply:
Good call, thanks for looking at this. Just pushed an update with your suggestion. 
It's cleaner and if one of the field names changes it won't break this test anymore.

Reply:
> @ryan-carroll-graylog If you have time to create the backport today, we can try to get that merged in time for the `beta-2` build tomorrow.

Sure thing @danotorrey , once the build succeeds I'll merge this and then backport it.